dlt/sources/rest_api/__init__.py

```diff
@@ -410,6 +410,16 @@ def _validate_config(config: RESTAPIConfig) -> None:
         auth = client_config.get("auth")
         if auth:
             auth = _mask_secrets(auth)
```

Comment:
Unrelated to this PR: there's not much point in this assignment, but I'll fix that in a follow up PR

Reply:
removed in e2f6c8cf